tools/src/test/scala/com/nvidia/spark/rapids/tool/qualification/AppFilterSuite.scala

```diff
@@ -302,6 +302,36 @@ class AppFilterSuite extends FunSuite {
       "3w", 2)
   }
 
+  test("full app name exact and 2-oldest-overall no match from filename") {
+    testFileSystemTimeAndStartAndAppFull(appsFullWithFsToTest, "2-oldest-overall", "nds", "nomatch",
+      "3w", 0)
+  }
+
+  test("full app name exact and 2-oldest-overall") {
+    testFileSystemTimeAndStartAndAppFull(appsFullWithFsToTest, "2-oldest-overall", "nds", "nds",
+      "3w", 2)
```

Comment:
this isn't verifying oldest vs newest

Reply:
created another tests which verifies the same by comparing the result appNames with expected appNames.